packages/core/src/node-utils/manifest.ts

```diff
@@ -62,9 +64,34 @@ export function patchManifest(cwd: string, data: Partial<Manifest>): boolean {
   return true;
 }
 
+export function createOrUpdateManifest(
```

Comment:
doesn't seem like a necessary function given what we had before.
As Arnaud said on one of his command, if we want to validate the File system when we are readying it then it seems better to keep using the same helpers as before and use io-ts on the retrieveManifest function.

Reply:
If it goes in the retrieveManifest, there's going to be a lot of warnings happening before the exit message,
<img width="1415" alt="Screenshot 2022-04-06 at 12 02 39" src="https://user-images.githubusercontent.com/7499570/161950815-bed148d2-1d16-48df-8b6e-b4c5194fd5fb.png">

Reply:
Why ?

The core when validating returns IO-TS errors or custom errors that are catch by the code that uses it no ?

Reply:
`retrieveManifest` is called a lot thought slice-machine, if we do the validation in it then we get a result like that

Reply:
or are you saying do it anyway?